src/VirtualClient/VirtualClient.Main/profiles/PERF-MEM-LMBENCH.json

```diff
@@ -1,42 +1,55 @@
 {
-    "Description": "LMbench Performance Workload",
-    "MinimumExecutionInterval": "00:10:00",
-    "Metadata": {
-        "RecommendedMinimumExecutionTime": "(4-cores)=04:00:00,(16-cores)=10:00:00,(64-cores)=16:00:00",
-        "SupportedPlatforms": "linux-x64,linux-arm64",
-        "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu"
+  "Description": "LMbench Performance Workload",
+  "MinimumExecutionInterval": "00:10:00",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "(4-cores)=04:00:00,(16-cores)=10:00:00,(64-cores)=16:00:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64",
+    "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu"
+  },
+  "Parameters": {
+    "CompilerName": "gcc",
+    "CompilerVersion": "10",
+    "CompilerFlags": "CPPFLAGS=\"-I /usr/include/tirpc\"",
```

Comment:
Which systems need these flags?

Reply:
ubuntu 22 both x64 and arm64 platforms require these flags.

Reply:
which systems are you testing on? I remember we have ubuntu22 data on lmbench

Reply:
I was testing on azure VMs ubuntu22 arm machines. Also facing same issue on cedarcrest machines.